src/main/js/EditorPage.jsx

```diff
@@ -112,11 +112,22 @@ SaveDialog.propTypes = {
 
 @observer
 class PipelineLoader extends React.Component {
-    state = {}
-    
+    constructor(props) {
+        super(props);
+        this.state = { scmId: 'github' };
```

Comment:
For backward compatibility as scmSource element is added to blueocean pipeline output recently. When we move off from org folder then we can clean it up.